plugins/nessus.rb

```diff
@@ -1017,7 +1017,7 @@ def is_scan_complete(scan_id)
         complete = false
         status = @n.scan_list
         status["scans"].each { |scan|
-        if scan["id"] == scan_id.to_i && scan["status"] == "completed"
+        if scan["id"] == scan_id.to_i && scan["status"] == "completed" || "imported"
```

Comment:
This is supposed to be a boolean evaluation... but if you're writing it this way, this is what happens:

```
2.1.5 :002 > a == "abc" || "123"
 => "123"
```

Do you mean:

```
2.1.5 :003 > a == "abc" || a == "123"
 => false
```